lib/IRGen/GenMeta.cpp

```diff
@@ -2691,6 +2688,11 @@ namespace {
       B.addInt(IGM.SizeTy, 0);
     }
 
+    void addValueWitnessTable() {
+      // The runtime fills in the value witness table for us.
+      B.addInt(IGM.SizeTy, 0);
```

Comment:
Why not `IGM.ValueWitnessTableTy`?

Reply:
There doesn't appear to be such a thing but elsewhere we use IGM.WitnessTablePtrTy. I'll use that.

Reply:
Seems reasonable to add as an alias just for self-documenting interests.